File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    const alpha = "abcdefghijklmnopqrstuvwxyz";
    let noSkipAlpha = [...alpha].filter((v) => !skip.includes(v));
    return [...s].map((v) => noSkipAlpha[(noSkipAlpha.indexOf(v) + index) % noSkipAlpha.length]).join("");
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alpha = "abcdefghijklmnopqrstuvwxyz"
    no_skip_alpha = [c for c in alpha if c not in skip]
    
    result = []
    for ch in s:
        i = no_skip_alpha.index(ch)
        shifted_char = no_skip_alpha[(i + index) % len(no_skip_alpha)]
        result.append(shifted_char)
    
    return ''.join(result)